src/query_parser/v3/parser.y

```diff
@@ -713,14 +722,34 @@ modifierlist(A) ::= modifierlist(B) OR term(C). {
     A = B;
 }
 
-expr(A) ::= ISEMPTY LP modifier(B) RP . {
+/////////////////////////////////////////////////////////////////
+// Single Tag - tag enclosed in curly braces
+/////////////////////////////////////////////////////////////////
+
+expr(A) ::= modifier(B) COLON LB single_tag(C) RB . {
+    if (!C) {
+        A = NULL;
+    } else {
+      // Tag field names must be case sensitive, we can't do rm_strdupcase
+        char *s = rm_strndup(B.s, B.len);
+        size_t slen = unescapen(s, B.len);
+
+        A = NewTagNode(s, slen);
+        QueryNode_AddChildren(A, C->children, QueryNode_NumChildren(C));
+
+        // Set the children count on C to 0 so they won't get recursively free'd
+        QueryNode_ClearChildren(C, 0);
+        QueryNode_Free(C);
+    }
+}
```

Comment:
Is this required? No tag list here.

Reply:
Yes, it is required. It is part of tag autoescaping, It existed in the previous code, see lines 752-772 on right column.